<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CentralBankWidget</class>
 <widget class="QWidget" name="CentralBankWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>1011</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Курсы валют</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="topLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="lastUpdateLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Укажите параметры ниже чтобы посмотреть курс конкретной валюты:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Буквенный код:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="charCodeLine"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>До какой даты:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="upToDateLine"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="graphButton">
         <property name="text">
          <string>Показать изменение курса валюты</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
